src/libtilt/shapes/soft_edge.py

```diff
@@ -8,7 +8,7 @@ def _add_soft_edge_single_binary_image(
 ) -> torch.FloatTensor:
     if smoothing_radius == 0:
         return image.float()
-    distances = ndi.distance_transform_edt(torch.logical_not(image))
+    distances = ndi.distance_transform_edt(torch.logical_not(image)) #TODO: This breaks if the input device is cuda
```

Comment:
```suggestion
    distances = ndi.distance_transform_edt(torch.logical_not(image))
```
nice catch, will make an issue

Reply:
#57 open